test/fuzz/CMakeLists.txt

```diff
@@ -3,6 +3,15 @@ include_directories(${PROJECT_SOURCE_DIR}/src)
 include_directories(${PROJECT_BINARY_DIR}/src)
 include_directories(${PROJECT_SOURCE_DIR}/src/box)
 
+# fiber.h may require libunwind.h. Add proper header search paths
+# and dependencies.
+function(add_libunwind_dependency target)
+  if (ENABLE_BACKTRACE AND ENABLE_BUNDLED_LIBUNWIND)
+    target_include_directories(${target} SYSTEM PRIVATE ${LIBUNWIND_INCLUDE_DIR})
+    add_dependencies(${target} bundled-libunwind bundled-libunwind-platform)
+  endif()
+endfunction()
```

Comment:
Same comment as for the unit tests.

Reply:
I verified visually that the fuzzers have an indirect dependency on `core` and removed the `add_dependencies()` directive. I built the project with fuzzers locally several times (with `make -j`) and everything looks fine. (All the builds were without system's `libunwind`.)